pkg/cmd/roachtest/test_test.go

```diff
@@ -315,19 +315,21 @@ func TestRunnerTestTimeout(t *testing.T) {
 		cpuQuota:  1000,
 		debugMode: NoDebug,
 	}
-	numTasks := 3
+
+	var tasksStarted atomic.Uint32
 	var tasksDone atomic.Uint32
 	test := registry.TestSpec{
 		Name:             `timeout`,
 		Owner:            OwnerUnitTest,
-		Timeout:          10 * time.Millisecond,
+		Timeout:          100 * time.Millisecond,
```

Comment:
Seems kinda arbitrary? Why not use a waitgroup to ensure all three tasks are started before testing the timeout in `runner.Run`? In that case, the actual `Timeout` value can be set artificially low.

Reply:
Updated, to use a channel to ensure tasks are started.